File: 30DaysOfCode/Day2/Operators.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Operators {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        double meal_cost = in.nextDouble();
        int tip_percent = in.nextInt();
        int tax_percent = in.nextInt();
        in.close();
        
        double totalCost;
        int output;
        
        totalCost = meal_cost*(100+tip_percent+tax_percent)/100;
        output = (int)Math.round(totalCost);
        
        System.out.println("The total meal cost is " + output + " dollars.");
    }
}

File: 30DaysOfCode/Day2/Operators.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    double meal_cost;
    cin >> meal_cost;
    int tip_percent;
    cin >> tip_percent;
    int tax_percent;
    cin >> tax_percent;
    
    double total_cost;
    int output;
    
    total_cost = meal_cost * (100 + tip_percent + tax_percent)/100;
    output = round(total_cost);
    
    cout << "The total meal cost is " << output << " dollars." << endl;
    return 0;
}